MultiModalCLF/MoodBoardMultiModalClassification.ipynb

```diff
@@ -0,0 +1,3617 @@
+        "\n",
+        "import torch\n",
+        "import torchvision\n",
+        "\n",
+        "from torch.utils.data import Dataset, DataLoader\n",
+        "from sklearn.model_selection import train_test_split\n",
+        "\n",
+        "import opendatasets as od\n",
+        "\n",
+        "from PIL import Image\n",
+        "import os\n",
+        "import json\n",
+        "from transformers import AutoModel, AutoTokenizer\n",
+        "from transformers import Trainer, TrainingArguments\n",
+        "from transformers.optimization import get_cosine_schedule_with_warmup\n",
+        "from transformers import BertModel, BertConfig\n",
+        "import transformers\n",
+        "\n",
+        "import accelerate\n",
+        "from torchvision import transforms\n",
+        "import torch.nn as nn\n",
+        "import torch\n",
+        "import csv\n"
+      ],
+      "metadata": {
+        "id": "XIHUXfB_f_CI"
+      },
+      "execution_count": 3,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "dataset_url = 'https://www.kaggle.com/datasets/trungit/wikiart25k'\n",
+        "od.download(dataset_url)"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "ibVhWj2rymvR",
+        "outputId": "e92c3d51-9b8f-4b85-b72c-f93a3939516c"
+      },
+      "execution_count": 4,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Please provide your Kaggle credentials to download this dataset. Learn more: http://bit.ly/kaggle-creds\n",
+            "Your Kaggle username: ruthleee\n",
+            "Your Kaggle Key: ··········\n",
+            "Downloading wikiart25k.zip to ./wikiart25k\n"
+          ]
+        },
+        {
+          "output_type": "stream",
+          "name": "stderr",
+          "text": [
+            "100%|██████████| 7.74G/7.74G [00:30<00:00, 275MB/s]\n"
+          ]
+        },
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "import os\n",
+        "\n",
+        "# function to delete incorrectly encoded files\n",
+        "\n",
+        "def delete_non_ascii_filenames(directory):\n",
+        "    count = 0\n",
+        "    for subdir, _, files in os.walk(directory):\n",
+        "        for file in files:\n",
+        "            try:\n",
+        "                file.encode('ascii')\n",
+        "            except UnicodeEncodeError:  # Filename is non-ASCII\n",
+        "                full_path = os.path.join(subdir, file)\n",
+        "                os.remove(full_path)  # Delete the file\n",
+        "                count += 1\n",
+        "                print(f\"Deleted file: {full_path}\")\n",
+        "    return count\n",
```

Comment:
How was this decision made to delete files with non ascii files? Would this not target specific artists and skew the dataset in any way?